.gitignore

```diff
@@ -16,6 +17,7 @@ server.key
 ######################
 /tests/Logs/*
```

Comment:
it's in there so we dont delete all files from people who havent moved yet..
In tests, logdir is also inside DATA_DIR, so in tests/data/logs, which is .gitignores as per tests/data*